verification/curator-service/api/test/users.test.ts

```diff
@@ -152,3 +152,29 @@ describe('PUT', () => {
             .expect(400);
     });
 });
+
+
+describe('DELETE', () => {
+    it('should delete a user', async () => {
+        const request = supertest.agent(app);
+        const userRes = await request
+            .post('/auth/register')
+            .send({ ...baseUser, ...{ roles: ['admin'] } })
+            .expect(200, /admin/)
+            .expect('Content-Type', /json/);
+        const res = await request
+            .delete(`/api/users/${userRes.body._id}`)
+            .expect(204);
+    });
+    it('cannot delete an nonexistent user', async () => {
+        const request = supertest.agent(app);
+        await request
+            .post('/auth/register')
+            .send({ ...baseUser, ...{ roles: ['admin'] } })
+            .expect(200)
+            .expect('Content-Type', /json/);
+        return request
+            .delete('/api/users/5ea86423bae6982635d2e1f8')
+            .expect(404);
+    });
+});
```

Comment:
two important tests: can it delete a user that isn't the requesting user, and does it correctly restrict a non-admin user from deleting a user?